src/libs/auth.ts

```diff
@@ -74,6 +76,8 @@ export const signOut = () => {
 };
 
 export const signOutAfterSessionTimeout = () => {
+  const user: User = getUser();
+  Ajax().Metrics.captureEvent(Events.userTimeoutLogout, { user });
```

Comment:
Here, I _do_ like the idea of firing the `captureEvent` first before calling `signOut`, that way when we look at the metrics, they will hopefully show up in the right order.  Then again, they're `Promise`s, so 🤷 .

Anyway, once again, think about the information you want to capture here (besides grabbing the whole `User` object) that is particularly relevant to a session timeout as opposed to a regular sign out.  We should still get the sign out data, so we should really think about session timeout here and what is exclusively relevant to these events.

Reply:
It might make sense to make a `UserMetrics` object that we can safely pass off to mixpanel with all the information we care about.

Reply:
Minor comment -- it would be cool to add some instrumentation around `reloadAuthToken` as well, since failures there may be contributing to the double-login bug (see my comment in [JIRA](https://broadworkbench.atlassian.net/browse/ID-557)).